frontend/src/components/Button.tsx

```diff
@@ -19,8 +19,8 @@ const StyledButton = styled.button<{
   box-shadow: ${(props) => (props.state === 'primary' ? `0px` : `2px 1000px 1px #fff inset`)};
   color: ${(props) => (props.state === 'primary' ? `white` : `#58B994`)};
   padding: ${(props) => (props.size === 'small' ? `7px 14px` : `16px 30px`)};
-  line-height: 1px;
-  white-space: nowrap;
+  white-space: normal;
+  cursor: pointer;
```

Comment:
just a heads up i helped y'all add this line so that the button will show a pointer when hovered over by the mouse cursor